package_bag/routines.py

```diff
@@ -0,0 +1,211 @@
+from .models import Bag
+from package_bag.serializers import BagSerializer
+
+from uuid import uuid4
+from shutil import move
+import bagit
+import bagit_profile
+import os
+from os import listdir, rename, remove, mkdir
+from os.path import join, splitext, basename
+from requests import post
+import tarfile
+import json
+
+from zorya import settings
+
+
+class DiscoverBags(object):
```

Comment:
```suggestion
class BagDiscoverer(object):
```
Semantically, it makes more sense to name these classes as nouns rather than verbs, since in an OO world they are things not functions.